app/src/main/java/de/westnordost/streetcomplete/quests/QuestsModule.kt

```diff
@@ -244,252 +245,253 @@ fun questTypeRegistry(
 
     10 to AddBenchBackrest(),
     11 to AddPicnicTableCover(),
+    12 to AddBbqCover(),
 
-    12 to AddBridgeStructure(),
+    13 to AddBridgeStructure(),
```

Comment:
note: you should not change numbers identifying other quests, see for example https://github.com/streetcomplete/StreetComplete/commit/c60c1bf8fbdbf677419b3b62e0dbd743bf0f8cc9

This numbers are supposed to be constant for each quest to keep presets saved as QR codes working

Reply:
156 is taken already, 157 seems free and never used

Reply:
oh, I thought they were for organizing the quests by priority, let me fix that quickly ^^'

Reply:
priority is decided by order of quests in list

have you used some documentation? (if yes, then it may need updating)

Reply:
I was following https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING_A_NEW_QUEST.md#add-the-quest-to-the-list-of-active-ones which left it to my imagination what those numbers were. "Ordered by priority" the number was the priority. Also "As this point" -> "At this point" :). Note that I force-pushed the branch, it's now number 157.

Reply:
@roptat Would you fancy opening another PR to improve that part in the documentation? Otherwise, I can do that.

Reply:
OK, I tried to write something in #4878